ansible/library/pulp_facts.py

```diff
@@ -48,14 +48,17 @@
         rpm_dependency_list += [rpm.split()[0] for rpm in stdout.splitlines()]
 
 # Remove any duplicates
-rpm_dependency_list = list(set(rpm_dependency_list))
+rpm_dependency_list = set(rpm_dependency_list)
+if 'python-twisted' in rpm_dependency_list:
+    rpm_dependency_list.remove('python-twisted')
+    rpm_dependency_list.add('python2-twisted')
```

Comment:
Just glancing at this I think this a bug in python-twisted's packaging. We should probably just have them fix it rather than work around it.